web-console/knikubevirt/Create-vm/create-vm.md

```diff
@@ -0,0 +1,156 @@
+
+As soon as the OS, Flavor, or Workload profile fields are modified, the “Create new template from configuration” checkbox will no longer be disabled.
+
+## Provision Source Field
+
+### ISO (this is not implemented yet, but is planned)
+Provision source tooltip content: Use an optical disk image that contains installation media for an operating system.
+
+![ISO](img/Step-1-basic-ISO.png)
+
+ISO enables the user to install an Operating System from existing installation media within ContainerDisk.
+
+![ISO selected](img/Step-1-basic-ISO-2.png)
+
+The user must select an ISO and then manually specify the OS, Flavor, and Workload Profile. None of those fields can be automatically pre-filled.
+
+
+### PXE
+Provision source tooltip content: Discover provisionable virtual machines over the network.
+
+![PXE](img/Step-1-basic-PXE-1.png)
+
+PXE enables the user to install an OS and configure it over a network. PXE depends on DHCP to find the PXE Server on an L2 Network. If one is detected, it will automatically be selected in the Networking step (see Networking section) and a note will be displayed below Provision Source.
+
+If a PXE-capable NIC cannot be found, one will be automatically created with the name “PXE” and selected by default.
+
+### URL
+Provision source tooltip content: An external URL to the .iso, .img, .qcow2 or .raw that the virtual machine should be created from.
+
+![URL](img/Step-1-basic-url.png)
+
+
+### Container
+Provision source tooltip content: Ephemeral virtual machine disk image which will be pulled from container registry.
+
+Container allows the user to provide an image from container registry.
+
+The user must provide a container image and then manually specify the OS, Flavor, and Workload Profile. None of those fields can be automatically pre-filled.
+
+![Container](img/Step-1-basic-container.png)
+
+### Attach Disk
+Provision source tooltip content: Select an existing disk that has been previously cloned or created.
+
+When selecting `Attach disk` the user will be prompted to select an available disk that has been previously cloned or created and made available in the Persistent Volume Claims. They must manually specify the OS, Flavor, and Workload Profile. None of those fields can be automatically pre-filled.
+
+![Attach-disk](img/Step-1-basic-attach-disk.png)
+
+![attach-disk-dropdown](img/Step-1-basic-attach-disk-dropdown.png)
+
+This list is populated by making a PVC available as a provision source for a virtual machine. This list is limited by the namespace it belongs to. These lists have the potential to be very long so a search field can help alleviate scrolling.
+
+![attach-disk-selected](img/Step-1-basic-attach-disk-selected.png)
+
+
+## Operating System Field
+
+Recommended minor version editing
+
+![OS](img/Step-1-basic-Operating-system-1.png)
+
+Only major version numbers are shown in the OS dropdown. The user selects one.
+
+![OS minor version](img/Step-1-basic-Operating-system-2.png)
```

Comment:
Nit: Minor version dropdown here should be 8.0 (or whatever's the latest) rather than 7.x. This applies to the next screen too.